store/iavlstore_test.go

```diff
@@ -144,6 +146,43 @@ func testNormal(t *testing.T) {
 	diskDb.Close()
 }
 
+func TestLoadIAVLVersion(t *testing.T) {
+	flushInterval = int64(5)
+	memDB := db.NewMemDB()
+	store, err := NewIAVLStore(memDB, 0, 0, flushInterval)
+	require.NoError(t, err)
+	var lastVersion int64
+	var s string
+	ch := make(chan int64)
+	go func() {
+		for i := int64(1); i <= flushInterval*10000; i++ {
+			s = strconv.FormatInt(i, 10)
+			store.Set([]byte("key"+s), []byte("value"+s))
+
+			// Broken Unpredictably
+			_, lastVersion, err = store.SaveVersion(nil)
+
+			// Works fine
+			// _, lastVersion, err = store.tree.SaveVersion()
+			ch <- lastVersion
```

Comment:
I want this test to only load the versions before the ones flushed out, i.e. only versions that satisfy the condition:
```golang
version % flushInterval == (flushInterval - 1)
```